pkg/apis/kops/validation/aws.go

```diff
@@ -107,3 +109,20 @@ func awsValidateAMIforNVMe(fieldPath *field.Path, ig *kops.InstanceGroup) field.
 	}
 	return allErrs
 }
+
+func awsValidateSpotDurationInMinute(fieldPath *field.Path, ig *kops.InstanceGroup) field.ErrorList {
+	allErrs := field.ErrorList{}
+	if ig.Spec.SpotDurationInMinutes != nil {
+		if *ig.Spec.SpotDurationInMinutes < 60 {
+			errDetails := fmt.Sprintf("cannot be lesser than 60")
```

Comment:
```suggestion
			errDetails := fmt.Sprintf("cannot be less than 60")
```

Reply:
Then should I change the greater than as well? @johngmyers

Reply:
"greater than" is correct. English is indeed inconsistent.